docs/reference/ml/df-analytics/apis/analysisobjects.asciidoc

```diff
@@ -145,7 +145,8 @@ include::{docdir}/ml/ml-shared.asciidoc[tag=lambda]
 include::{docdir}/ml/ml-shared.asciidoc[tag=dependent-variable]
 +
 --
-The data type of the field must be numeric or boolean.
+The data type of the field must be numeric (`integer`, `short`, `long`, `byte`), 
```

Comment:
Not sure this is a good place but it should be mentioned somewhere in the docs that the numbers should represent categories (so should be logically categorical e.g: 0 - cat, 1 - dog, 2 - horse) rather than arbitrary numbers

Reply:
I made a note and put the info to the right place.